src/views/virtualmachines/details/tabs/configuration/details/DetailsSection.tsx

```diff
@@ -133,24 +136,38 @@ const DetailsSection: FC<DetailsSectionProps> = ({ instanceTypeVM, vm, vmi }) =>
               />
             )}
             <VirtualMachineDescriptionItem
-              messageOnDisabled={t(
-                'CPU and Memory can not be edited if the VirtualMachine is created from InstanceType',
-              )}
+              descriptionHeader={
+                <SearchItem id="cpu-memory">
+                  {vm?.spec?.instancetype ? t('InstanceType') : t('CPU | Memory')}
```

Comment:
I think !isEmpty(vm?.spec?.instancetype) would be better. you can also put it in const since you're using this 2 times in this file :)